scripts/lib-franklin.js

```diff
@@ -366,13 +366,43 @@ export function decorateSupScript(string, result = []) {
   return result;
 }
 
+export function getInfo() {
+  const [, country, language] = window.location.pathname.split('/');
+  return {
+    country,
+    language,
+    productDB: new URL('/products.json', window.location).pathname,
+    productSupport: new URL(`/${country}/${language}/product-support`, window.location).pathname,
+    queryIndex: new URL(`/${country}/${language}/query-index.json`, window.location).pathname,
+  };
+}
+
+export function adjustAssetURL(asset) {
+  if (asset?.URL) {
+    const url = new URL(asset.URL, window.location);
+    if (url.hostname.endsWith('-mammotome--hlxsites.hlx.page')
+      || url.hostname.endsWith('-mammotome--hlxsites.hlx.live')
+      || url.hostname === 'localhost') {
+      asset.URL = url.pathname;
+    }
+  }
+  return asset;
+}
+
 export async function getProductDB() {
   if (!window.productDB) {
-    const resp = await fetch('/products.json?limit=10000');
+    const { productDB } = getInfo();
+    const resp = await fetch(`${productDB}?limit=10000`);
     if (!resp.ok) {
       throw new Error(`${resp.status}: ${resp.statusText}`);
     }
     window.productDB = await resp.json();
+    if (window.productDB.ProductAsset?.data) {
```

Comment:
in the same vein, if we like arrays and functional, we could make this slightly more maintainable:

```
    const adjustData = (field) => {
      if (window.productDB[field]?.data) {
        window.productDB[field].data = window.productDB[field].data.map(adjustAssetURL);
      }
    };

    ['ProductAsset', 'eIFU'].forEach(adjustData);
  }
  
  return window.productDB;
```